src/test/ui/loop-break-value-no-repeat.stderr

```diff
@@ -3,6 +3,10 @@ error[E0571]: `break` with value from a `for` loop
    |
 22 |         break 22 //~ ERROR `break` with value from a `for` loop
    |         ^^^^^^^^ can only break with a value inside `loop`
+help: instead, use `break` on its own without a value inside this `for` loop
+   |
+22 |         break //~ ERROR `break` with value from a `for` loop
```

Comment:
This second "by example" version of the error doesn't seem like it's providing the user with additional information. Would it be possible to change this PR to just add the additional `help` message, without the `break`-only example?

Reply:
I find that when suggesting _removal_ of text, the inline suggestion style lends itself to confusion more easily. What do you think?

Reply:
I find this particular line is confusing-- I don't think it adds new information, the `ERROR` pattern on it is misleading since there would be no error in this case (since it's the corrected code):

```rust
22 |         break //~ ERROR `break` with value from a `for` loop
```

Reply:
But that's just a comment, as far as `rustc` is concerned. If I change the test file to have that comment one line below, it would look like this:

```
error[E0571]: `break` with value from a `for` loop
  --> $DIR/loop-break-value-no-repeat.rs:22:9
   |
22 |         break 22
   |         ^^^^^^^^ can only break with a value inside `loop`
help: instead, use `break` on its own without a value inside this `for` loop
   |
22 |         break
   |         ^^^^^
```

instead of 

```
error[E0571]: `break` with value from a `for` loop
  --> $DIR/loop-break-value-no-repeat.rs:22:9
   |
22 |         break 22
   |         ^^^^^^^^
   |         |
   |         can only break with a value inside `loop`
   |         help: use `break` without a value: `break`
```

Reply:
Ah, okay. I personally still prefer the first one, but I'll defer to your judgement.